content/docs/1.6.0/monitoring/prometheus-and-grafana-setup.md

```diff
@@ -61,9 +63,26 @@ Follow instructions in [Prometheus Operator - Quickstart](https://github.com/pro
       - port: manager
     ```
 
-    Longhorn ServiceMonitor has a label selector `app: longhorn-manager` for selecting Longhorn backend service.
+#### Install Longhorn ServiceMonitor with Helm
+
+1. Modify the YAML file `longhorn/chart/values.yaml`.
+
+    ```yaml
+    metrics:
+      serviceMonitor:
+        # -- Setting that allows the creation of a Prometheus ServiceMonitor resource for Longhorn Manager components.
+        enabled: true
+    ```
+
+1. Create a ServiceMonitor for Longhorn manager with Helm
+
+    ```bash
+      helm upgrade longhorn longhorn/longhorn --namespace longhorn-system -f values.yaml
+    ```
 
-    Longhorn ServiceMonitor is included in the Prometheus custom resource so that the Prometheus server can discover all Longhorn manager pods and their endpoints.
+Longhorn ServiceMonitor has a label selector `app: longhorn-manager` for selecting Longhorn backend service.
+
+Longhorn ServiceMonitor is included in the Prometheus custom resource so that the Prometheus server can discover all Longhorn manager pods and their endpoints.
```

Comment:
```suggestion
Longhorn ServiceMonitor is included in the Prometheus custom resource. This setup allows the Prometheus server to discover all Longhorn Manager pods and their respective endpoints.

You can use the label selector `app: longhorn-manager` to select the longhorn-backend service, which points to the set of Longhorn Manager pods.
```

Reply:
I switched the sentences for better flow. I also added a description of the backend service to make the connection between the two points clearer.

Reply:
Modified. Thanks